### DISCUSSION TITLE
Next Meetup speakers and themes

### DISCUSSION THREAD

**Discussion by relu91** [w3c-cg/wot-cg#13] [Ideas]
This thread is about sharing ideas on the next topics for Community group meetups. In meetings and meetups, there were ideas about what topics would be. We can use this place to record and discuss them. If you are interested to speak or there are some important topics about WoT that should be discussed you have the opportunity to propose them here. 

Current project and speakers' ideas (no particular order and not yet confirmed):

- [x] @JKRhb: [DartWoT](https://pub.dev/packages/dart_wot) [RIoT OS](https://www.riot-os.org/) and WoT (🎉**Confirmed**: [23/02/2023 ])
- [x] @netzoio @miguelrk: WoT API platform (🎉**Confirmed**: [15/12/2022 at 4:00 P.M. CET](https://www.w3.org/events/meetings/f49cc19b-66a6-4d4c-a9c1-0ca327908486))
- [x] @robwin [Magenta Smart Home](https://www.smarthome.de/) (🎉**Confirmed**: [1/12/2022 at 4:00 P.M. CET](https://www.w3.org/events/meetings/07d67134-f9d0-4c8a-ba1e-24f25babef8c#general))
- [x] Schaeffler (🎉**Confirmed**: [11/05/2023 at 4:00 P.M. CET](https://www.w3.org/events/meetings/c51bb82c-70ae-4b77-bc58-40bd80f73fed/)
- [x] Siemens and Microsoft (🎉**Confirmed**: [04/04/2023 at 4:00 P.M. CET](https://www.w3.org/events/meetings/322db7e4-1a9f-44c6-8ddf-af601043dfb9/)
- [x] SICK AG (🎉**Confirmed**: [23/03/2023 at 4:00 P.M. CET](https://www.w3.org/events/meetings/e72912b5-3087-440e-950e-886acbab645d/) 
- [x] IntellIoT (🎉**Confirmed**: [23/03/2023 at 4:00 P.M. CET](https://www.w3.org/events/meetings/a15d3d7d-7815-4da5-8233-01aeb62ee227/) 
- [x] @benfrancis: [Webthings](https://webthings.io/gateway/), [Krellian](https://krellian.com/)
- [x] @fexpal [SayWoT!](https://www.evosoft.com/en/digitalization-offering/saywot/)
- [x] @ondrejtomcik [SayWoT! and Smart Edge Connector]: Planned for September-October
- [ ] @relu91 [Zion](https://zion.vaimee.com/): TDD implementation (or even an event with all TDD implementations with @AndreaCimminoArriaga [wot-hive](https://github.com/oeg-upm/wot-hive) @farshidtz [tiny-iot](https://github.com/TinyIoT/thing-directory))
- [ ] @egekorkan  [Thing Description Playground](http://playground.thingweb.io)/ [WoT API Development Environment (WADE)](https://github.com/tum-esi/wade) [Shadow Thing](https://github.com/tum-esi/shadow-thing) [Web of Things Test Bench](https://github.com/tum-esi/testbench)
- [ ] [k-toumura](https://github.com/k-toumura) [Node-Red Node generator](https://github.com/node-red/node-red-nodegen): Currently scheduled for after TPAC
- [x] @fexpal [ediTDor](https://eclipse.github.io/editdor/)
- [x] @danielpeintner [thingweb.node-wot](https://github.com/eclipse/thingweb.node-wot)
- [ ] @lu-zero [wot-rust](https://github.com/sifis-home/wot-td) next features 
- [ ] Echonet Lite Web API: Under discussion via @ashimura 
- [x] @mmccool @sebastiankb When the charter proposal is ready: Charter feedback or presentation 
- [x] @AndreaCimminoArriaga https://cogito-project.eu/ - digital twins (🎉**Confirmed**: [14/02/2024 at 4:00 P.M. CET](https://www.w3.org/events/meetings/1529773f-380c-4536-9504-9cb8c3d35bbf/)
- [ ] @hidetak FirestoreBinding - we are evaluating whether to incorporate the intervention with node-wot and/or a protocol binding dedicated meet-up.

Current "theme" ideas:
- DigitalTwins ([Past](https://github.com/w3c/wot-cg/blob/main/Presentations/2022/2022-10-WoTCG-Meetup1-AguzziKorkan.pdf))
- Edge Computing ([Past](https://github.com/w3c/wot-cg/blob/main/Presentations/2022/2022-12-WoTCG-Meetup3-AguzziKorkan.pdf))
- Smart Home ([Past](https://github.com/w3c/wot-cg/blob/main/Presentations/2022/2022-12-WoTCG-Meetup2-AguzziKorkan.pdf))

If you have ideas for speeches or want to confirm your participation, please use the thread or contact the chairs via email.

**Comment by relu91** [comment]
📢**Announcement**: two meetups confirmed:
>- @netzoio @miguelrk: WoT API platform (🎉**Confirmed**: [15/12/2022 at 4:00 P.M. CET](https://www.w3.org/events/meetings/f49cc19b-66a6-4d4c-a9c1-0ca327908486))
>- @robwin [Magenta Smart Home](https://www.smarthome.de/) (🎉**Confirmed**: [1/12/2022 at 4:00 P.M. CET](https://www.w3.org/events/meetings/07d67134-f9d0-4c8a-ba1e-24f25babef8c#general))

**Comment by relu91** [comment]
🎉 **Update**: @JKRhb speaker confirmed for the 23rd of February

**Comment by egekorkan** [comment]
Update: 
- [Schaeffler](https://www.schaeffler.com/en/) for the 11.05  
- 04 April 4pm: Siemens and Microsoft
- 27 April : Simon Mayer from University of St. Gallen. 

Once the titles of the presentations are known, calendar entries will be created.

**Comment by egekorkan** [comment]
In the end of September, we will have Sy from seam.co 🥳

**Comment by egekorkan** [comment]
So after TPAC, we have the following meetups planned, in no particular order:
- Seam.co (@sybohy)
- Constrained Devices and WoT (@lu-zero, @JKRhb , @jyhi, @Citrullin)
- WoT Bindings in Practice (@relu91 , @danielpeintner , @egekorkan)
- LabThings (@rwb27)

We will have a week of break after TPAC so these meetups can start after 18th of September.

**Comment by relu91** [comment]
Ideas for 2025 meetups:
- Dashjoin (from wot week) ✅
- Thing Model and their applications (tentative name) 
- Using Web of Things to track assets (@relu91 - with Archeion)
- HiveOT ✅
- TUM Hololens and WoT ✅
- Web of Things for AI agents (LMOS) ✅

**Comment by egekorkan** [comment]
The ideas (some copied from above):

- Thing Model and their applications (tentative name, Pedram)
- Using Web of Things to Track Assets (Cris)
- OPC UA Connectivity and Binding (Sebastian and Erich)

**Reply by egekorkan** [reply]
- WoT with Jelly for RDF Streaming by NeverBlink: https://jelly-rdf.github.io/dev/
- https://www.mbbm.de/mueller-bbm-gruppe/mueller-bbm-vibroakustik-systeme-gmbh/

**Comment by egekorkan** [comment]
- [x] Thing Models and Wattsense: Ege and Mohamed Zenadi
  - Date Proposal: October?
- [x] Thing Model Catalog, EdiTDor, Thing Models for IoT Products (Ege, Pedram)
- [ ] Using Web of Things to Track Assets (Cris) (to be done later)
- [x] Decentralized Oracle System for IoT Data (Ivan Zyrianoff) https://trustchain.ngi.eu/zonia-mc/
  - Date Proposal: 11.09.2025
- [x] OPC UA Binding (Sebastian and Erich)
  - Date Proposal: October
- [ ] BDI Agents Mapping to WoT (Roman Binkert) -> Joint with WebAgents CG. Agreed, but no date yet
  - Date Proposal: TBD
- [ ] OPC UA Connectivity and Prosys OPC Forge (Contact to be found)
- [ ] Manageable Affordances -> Recap of the work so far, probably towards winter
- [ ] GSoC at Thingweb Results -> UI WoT and TestThings
  - Date Proposal: October
- [x] WoT and BACnet systems (Dogan Fennibay) -> Contacted
- [ ] Update from SICK AG -> contacted
- [ ] An online panel-like meetup where previous meetup presenters talk -> Can be even streamed like a TV show panel. One can be industry, one can be research, one open source. "WoT's up" can be a name for it.
